### PR TITLE
Edit screen approvement

### PR DESCRIPTION
**Pull request by brianteeman**
I reverted the language changes for the fieldsets. To keep it simple for now we are just re-using existing strings

### CODE REVIEW

File: administrator/components/com_content/forms/article.xml [hunk 1]
```diff
@@ -244,9 +243,9 @@
 
 	</fieldset>
 
-	<fields name="attribs">
+	<fields name="attribs" label="COM_CONTENT_ATTRIBS_FIELDSET_LABEL">
```
Comment (bembelimen) [inline administrator/components/com_content/forms/article.xml:246]: Do we need a label for this tag? Is this used anywhere?

Comment (brianteeman) [reply administrator/components/com_content/forms/article.xml:246]: It wasn't that I added this label it was always there and my revert of your changes brought it back. I don't see where it is used either so can leave it on the basis it was always there or I can remove it.

Comment (bembelimen) [reply administrator/components/com_content/forms/article.xml:246]: TBH I don't care :D Let's not change it.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**